.github/workflows/build-pr-preview.yml

```diff
@@ -17,10 +17,11 @@ jobs:
       COMPONENT_NAME: bonita
       COMPONENT_BRANCH_NAME: ${{ github.head_ref }}
     steps:
-      - name: Build and publish PR preview
-        uses: bonitasoft/bonita-documentation-site/.github/actions/build-and-publish-pr-preview/@master
+      - name: Build preview
```

Comment:
types should be removed in the pull_request configuration, we needed this configuration for surge and we are not running surge anymore